src/test/suite/fixtures/recorded/navigationMap/chuckThirdCarHarpPastSecondCarWhaleTakeWhale.yml

```diff
@@ -0,0 +1,47 @@
+spokenForm: chuck third car harp past second car whale take whale
+languageId: plaintext
+command:
+  actionName: remove
+  partialTargets:
+    - type: range
+      start:
+        type: primitive
+        selectionType: token
+        modifier: {type: subpiece, pieceType: character, anchor: 2, active: 2, excludeAnchor: false, excludeActive: false}
+        mark: {type: decoratedSymbol, symbolColor: default, character: h}
+      end:
+        type: primitive
+        selectionType: token
+        modifier: {type: subpiece, pieceType: character, anchor: 1, active: 1, excludeAnchor: false, excludeActive: false}
+        mark: {type: decoratedSymbol, symbolColor: default, character: w}
+      excludeStart: false
+      excludeEnd: false
+  extraArgs: []
+initialState:
+  documentContents: hello world whatever
+  selections:
+    - anchor: {line: 0, character: 20}
+      active: {line: 0, character: 20}
+  marks:
+    default.h:
+      start: {line: 0, character: 0}
+      end: {line: 0, character: 5}
+    default.w:
+      start: {line: 0, character: 6}
+      end: {line: 0, character: 11}
+finalState:
+  documentContents: herld whatever
+  selections:
+    - anchor: {line: 0, character: 14}
+      active: {line: 0, character: 14}
+  marks:
+    default.h:
+      start: {line: 0, character: 0}
+      end: {line: 0, character: 2}
+    default.w:
+      start: {line: 0, character: 2}
+      end: {line: 0, character: 5}
```

Comment:
This behaviour is what we agreed upon in discussion: although technically the whale token has joined with the harp token, we preserve only the part of whale that remains; we don't join it with harp